sdcm/stress_thread.py

```diff
@@ -63,6 +64,8 @@ def run(self) -> None:
 
                     if pattern.search(line):
                         event.add_info(node=self.node, line=line, line_number=line_number).publish()
+                        if event.severity == Severity.CRITICAL and not self.stop_test_on_failure:
```

Comment:
Do we need same for scylla-bench?

Reply:
I'm not sure we have critical log events in scylla-bench.

I'll check tomorrow if we have any

Reply:
AFAIR we can get critical event when stress command fails

Reply:
scylla-bench only has this log error:
```
ScyllaBenchLogEvent.add_subevent_type("ConsistencyError", severity=Severity.ERROR, regex=r"received only")
```
so no criticals, and no need for implementing there for now.

Reply:
See this run: https://argus.scylladb.com/test/3190b0e9-a489-4e19-b855-32dc837b5431/runs?additionalRuns[]=e9fb4ce4-bec9-4429-995e-fc13f77a27bb

Reply:
as I said it's not log event, and we might have an issue with s-b logic that we should look at on it's own issue/PR